<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="Default test"
           tests="1"
           failures="1"
           errors="0"
           skipped="0"
           time="78.786">
  <properties />
            <testcase name="app_search_and_open_doc" time="78.786" classname="com.project.tests.SanityProdAppRamatGan">
                <failure type="java.io.IOException"
                  message="Destination &apos;\\OU1900TREHAG\data\Ramat-Gan\Projects RG\CityApp\images&apos; directory cannot be created"
              >
<![CDATA[java.io.IOException: Destination '\\OU1900TREHAG\data\Ramat-Gan\Projects RG\CityApp\images' directory cannot be created
  org.apache.commons.io.FileUtils.copyFile(FileUtils.java:1085)
  org.apache.commons.io.FileUtils.copyFile(FileUtils.java:1038)
  com.framework.utility.GeneralUtility.printScreen(GeneralUtility.java:498)
  com.project.tests.SanityProdAppRamatGan.app_search_and_open_doc(SanityProdAppRamatGan.java:129)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.lang.reflect.Method.invoke(Method.java:498)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
  org.testng.TestRunner.privateRun(TestRunner.java:648)
  org.testng.TestRunner.run(TestRunner.java:505)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
  org.testng.SuiteRunner.run(SuiteRunner.java:364)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
  org.testng.TestNG.runSuites(TestNG.java:1049)
  org.testng.TestNG.run(TestNG.java:1017)
  org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
  org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
  org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
      </failure>

          </testcase>
  
  
  
</testsuite>
